fdbclient/SpecialKeySpace.actor.cpp

```diff
@@ -2552,14 +2552,18 @@ Future<Optional<std::string>> DataDistributionImpl::commit(ReadYourWritesTransac
 					                                           iter->value().second.get().toString());
 				}
 			} else if (iter->range() == singleKeyRange(rebalanceIgnoredKey)) {
-				if (iter->value().second.get().size())
-					msg =
-					    ManagementAPIError::toJsonString(false,
-					                                     "datadistribution",
-					                                     "Value is unused for the data_distribution/rebalance_ignored "
-					                                     "key, please set it to an empty value");
-				else
-					ryw->getTransaction().set(rebalanceDDIgnoreKey, LiteralStringRef("on"));
+				ValueRef val = iter->value().second.get();
```

Comment:
Is this only for when someone tries to write directly to the systemkey instead of using the `rebalance` CLI?

Reply:
Yes. One use case of special key space is to use pure `set` `get` operations to manage the cluster.